lib/orm/schema.js

```diff
@@ -256,10 +256,16 @@ export default class Schema {
 
   /**
     Returns the first model in the database that matches the key-value pairs in `attrs`. Note that a string comparison is used.
+    Also remeber that attribute names are dasherized (for attributes with more than 1 word, like "createdAt" becomes "created-at")
```

Comment:
Hm – I don't think this is true. I think maybe somewhere else a dasherized compound word must have been used.

Could you try with a new app that uses a model with a compound attribute name, and see if it's the case?

Reply:
Hi Sam,

I'll do that. Might take me a little while to setup and test that, but I'll get back when I do!